languages/python/sqlalchemy-oso/sqlalchemy_oso/partial.py

```diff
@@ -78,8 +153,28 @@ def translate_and(expression: Expression, session: Session, model, get_model):
 
     return expr
 
-
 def translate_isa(expression: Expression, session: Session, model, get_model):
+    """Translate an Isa operation. (matches keyword)
```

Comment:
```suggestion
    """Translate an Isa operation. (``matches`` keyword)
```

Unsure if it's supposed to be 1 or 2 backticks, but looks like you're using 2 everywhere else.